docs/03-explore.md

```diff
@@ -39,11 +39,16 @@ launched, you can execute the following:
 
     oc logs deployments/cluster-version-operator -n openshift-cluster-version > operatorlog.txt
 
-The operator's log is **extremely** long, so it is recommended that you redirect it to a file instead of trying to look at it directly with the `logs` command.
+The operator's log is **extremely** long, so it is recommended that you
+redirect it to a file instead of trying to look at it directly with the
+`logs` command.
 
-# Exploring RHEL CoreOS
-The OpenShift 4 cluster is made of hosts that are running RHEL CoreOS.
-CoreOS is a container optimized, hardened, minimal footprint
+# Exploring Red Hat CoreOS
+** MASTERS ARE NO LONGER PUBLICLY ACCESSIBLE BY DEFAULT **
+** NEED TO ADD ADDITIONAL DOCS FOR ACCESS **
```

Comment:
These are not displayed **bold** seen in https://github.com/thoraxe/training/blob/beta2/docs/03-explore.md .
> NEED TO ADD ADDITIONAL DOCS FOR ACCESS

Additional "DOCS"? Is it your todo item? Or do you want to mean additional "bastion"?

Reply:
This is a ToDo. You all reviewed this PR too early :)

Reply:
Now fixed, although ugly and messy.